doc/installation.rst

```diff
@@ -3,58 +3,66 @@
 Installing Glue
 ===============
 
-There are several ways to install Glue on your computer. We recommend one of
-the first two options, which greatly simplify of the (sometimes tricky) task of
-installing Qt, Matplotlib, and other libraries that Glue relies on.
+There are several ways to install Glue on your computer:
 
-.. note :: Glue support with Python 3 is currently experimental.
+Recommended: Anaconda Python Distribution
+-----------------------------------------
 
-Easiest Option (Recommended for Mac users)
-------------------------------------------
+**Platforms:** MacOS X, Linux, and Windows
 
-Mac users with OS X >= 10.7 can download Glue as a `standalone program
-<http://mac.glueviz.org>`_.
+We recommend using the `Anaconda <http://continuum.io/downloads.html>`_
+Python distribution from Continuum Analytics (or the related Miniconda
+distribution). Anaconda includes all of Glue's main dependencies.
 
-Easy Option (Recommended for Windows, Unix users)
--------------------------------------------------
+Once Anaconda or Miniconda are `installed
+<http://continuum.io/downloads.html>`_ for your operating system, you
+can install glue using::
 
-We recommend using the `Anaconda
-<http://continuum.io/downloads.html>`_ Python distribution from
-Continuum Analytics. Anaconda includes all of Glue's main dependencies.
+    conda install glueviz
```

Comment:
sidenote: `conda install glueviz` currently fails on python 3. I assume that's because conda still thinks glue isn't python-3 compatible? We should:

 * Add python 3 to the classifiers list on setup.py
 * Tell someone from Continuum that they can relax that restriction